docs/event-types.mdx

```diff
@@ -447,6 +447,9 @@ Enabling this setting will cause your event types to be statically served on **s
 - **Make Public:** Whether or not the Event Catalog will be publicly accessible from **svix.com**.
 - **Display Name:** _Required to make your Event Catalog public._ The display name will be shown in the heading of the published page. It should be the name of your company or product.
 
+### Custom domains
+If you want to show the Event Catalog from your own custom domain, such as **webhooks.example.com**, rather than the default **svix.com**, you can! [Contact us](https://www.svix.com/contact/) and we'll help you set it up. Pro and Enterprise tiers only.
+
```

Comment:
```suggestion
Customers on the Professional and Enterprise tiers can also have the Event Catalog served from their own domain, such as **webhooks.example.com** rather than the default **www.svix.com**. Please [contact us](https://www.svix.com/contact/) and we'll help you set it up.

```